src/Helpers/helperHelperText.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if (!function_exists('outputAriaForHelpterText')) {
```

Comment:
"Helpter" ...? I'm assuming that's a typo (you have it everywhere it's called too)

Reply:
yup, fixed